product/consul.go

```diff
@@ -69,9 +84,20 @@ func consulRunners(cfg Config, api *client.APIClient) ([]runner.Runner, error) {
 	// try to detect log location to copy
 	if logPath, err := client.GetConsulLogPath(api); err == nil {
 		dest := filepath.Join(cfg.TmpDir, "logs/consul")
-		logCopier := runner.NewCopier(logPath, dest, cfg.Since, cfg.Until, nil)
+		logCopier := runner.NewCopier(logPath, dest, cfg.Since, cfg.Until, cfg.Redactions)
 		runners = append([]runner.Runner{logCopier}, runners...)
 	}
 
 	return runners, nil
 }
+
+// getDefaultConsulRedactions returns a slice of default redactions for this product
+func getDefaultConsulRedactions() []*redact.Redact {
```

Comment:
The `getDefault` part of the name isn't needed here or in other products where this is used. We can name it after the kind of data it returns, e.g. `consulRedactions`, just like `consulRunners`. In this way it's like storing the data in a var named `consulRedactions`, only we call the function to return it. We could keep default in the name, but then it would differ from the `<product>Runners` naming, so let's keep it similar.

Reply:
That totally makes sense. Fixed!